src/AutoChessSharp.Core/Controller/GameRunner.cs

```diff
@@ -64,13 +64,13 @@ public bool SetStorePieces(string path)
     /// </summary>
     /// <param name="path"></param>
     /// <returns>Piece collection</returns>
-    private List<Piece> GetStorePiecesDB(string path)
+    private List<AutoChessPiece> GetStorePiecesDB(string path)
```

Comment:
Piece class has been renamed to autochesspiece to better signify the use and emphasis of the concrete class
Implying only pieces from this class are allowed in the autochess game
but piece interface IPiece still provided, extension and development should start from there